### ISSUE TITLE
layout-ncol issue with image layout and captions

### ISSUE THREAD

**Issue by jgunstone** [quarto-dev/quarto-cli#8785]
### Bug description

- images aren't centred correctly above text
- unwanted and unspecified (a), (b) ... etc. fig identifiers being added to images

### Steps to reproduce

```qmd
---
document_description: A description of the document that is important
footer-logo: logo.png
title-page: title-page.pdf
---

# Images

## A Red Circle

![Some red circle](./media/red-dot.png){width=30mm}

Above we can see an image of a red circle.

{{< pagebreak >}}

## Radiator Widths

::: {layout-ncol=3 layout-valign="bottom"}
![A logo](https://placehold.co/50x40.png){height=30mm}

![Another logo](https://placehold.co/50x40.png)

![Look at that, another logo!](https://placehold.co/50x40.png)
:::
```

### Expected behavior

behaviour in v1.4.466
![image](https://github.com/quarto-dev/quarto-cli/assets/21370980/829fd891-025d-4fd3-9bbf-0e5b34022380)

### Actual behavior

behaviour in v1.4.550
![image](https://github.com/quarto-dev/quarto-cli/assets/21370980/7681b536-df79-46d1-99c2-4d38681dba0a)

with public images
![image](https://github.com/quarto-dev/quarto-cli/assets/21370980/59a04e45-8d70-4964-948a-5ebe8390cfe0)

### Your environment

wsl, Ubuntu22.04, VS Code

### Quarto check output

```bash
Quarto 1.4.550
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.58.3: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.550
      Path: /home/jovyan/miniforge3/envs/document-issue-dev/lib/python3.11/site-packages/quarto_cli/quarto-1.4.549/bin

[✓] Checking tools....................OK
      TinyTeX: v2024.02
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /home/jovyan/.TinyTeX/bin/x86_64-linux
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.13 (Conda)
      Path: /home/jovyan/miniforge3/bin/python
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with conda install jupyter

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

**Comment by cscheid** [comment]
The presence of (a), (b), and (c) subfigure indicators is a difference between HTML and PDF behaviors, but I don't think it's a bug; we can't use LaTeX's mechanisms without creating them.

**Comment by jgunstone** [comment]
@mcanouil - just updated with placeholder images... 
@cscheid - is there a way to switch of the subfig indicators?

also the layout not being centred.... ?

**Comment by cscheid** [comment]
I've bisected the bug to 71de80cc56a46d067f626328a4794b223b4c0dc5

**Comment by cscheid** [comment]
Fixed on `main` and backported to 1.4.

**Comment by jgunstone** [comment]
thank you
